src/objects/config.ts

```diff
@@ -7,8 +7,10 @@ let config: RequiredGraphConfig | null = null
 
 const defaults = {
   animationDuration: 200,
+  nodeRenderKey: (node) => `${node.id},${node.kind},${node.start_time},${node.end_time},${node.state_type},${node.label}`,
```

Comment:
I thought about using a hashing function and just hashing the node itself. But I feel like this is probably much faster even if its more brittle. You have to make sure to add any properties here that effect the rendered node.

Reply:
Could probably improve this quite a bit by having this just return an array of keys or something. For now this is good enough.